include/indexstore/IndexStoreCXX.h

```diff
@@ -42,6 +44,28 @@ class IndexStoreLibrary {
   const indexstore_functions_t &api() const { return functions; }
 };
 
+class IndexStoreCreationOptions {
```

Comment:
IMO calling this options makes things pretty confusing. Should we just grab `PathRemapper`?

Reply:
Hmm but the C API is `creation_options_add_prefix_mapping`, I think it makes sense to have this in case more options are added, right?

Reply:
I don't think the intention is that this has to match the C API. Right now the equivalent IndexStoreCXX.h in LLVM just has the PathRemapper passed. Maybe that should have had options as well, but I believe we don't care about stability there so 🤷

Reply:
Ah I didn't add `PathRemapper` because that's in llvm/Basic and we don't need all of the logic that's there.